### PR TITLE
Fixes Remove `focusin` and `focusout` polyfill from vendor/ jquery-validate

### LINKED ISSUE #2419: Remove `focusin` and `focusout` polyfill from vendor/ jquery-validate
### Description
Inside vendor/js/jquery-validate/jquery.validate.js is some code  that "provides cross-browser focusin and focusout events"

Support for these events is pretty good now, so this code is no longer necessary

### Evidence / Screenshot (if possible)
https://developer.mozilla.org/en-US/docs/Web/API/Element/focusin_event

### Expectation
- Everything continues to work the same as before.
- Explicitly test the form validation code 
- Fixing this will remove the console warning "jquery-migrate.js:41 JQMIGRATE: jQuery.browser is deprecated"

### PR DESCRIPTION
**Pull request by cbolgiano**
Closes #2419

### Technical
Updated `jquery-validate`'s version from `1.6` to `1.19.1`.

### Testing
1. Open developer console.
2. Check the console to make sure that the `jquery-migrate.js:41 JQMIGRATE: jQuery.browser is deprecated` is gone for the `jquery.validate.js` file.
3. Login as admin
4. Click on add a book.
5. Try to add a book without any of the fields filled out.  See that the required fields turn red still and prevent you from continuing.
6. Fill in those fields in now and when on the next screen delete the title and try to submit the form.  You should see the required Title field turn red and prevent you from continuing.

### Evidence
Add a book form:
required1

Add more information form:
required2

Developer Console:
dev_console

### CODE REVIEW

File: openlibrary/plugins/openlibrary/js/index.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import '../../../../vendor/js/jquery-ui/jquery-ui-1.12.1.min.js';
 import '../../../../vendor/js/colorbox/1.5.14.js';
 // jquery.form#2.36 not on npm, no longer getting worked on
 import '../../../../vendor/js/jquery-form/jquery.form.js';
-// jquery-validate#1.6 not on npm
+// jquery-validate#1.19.1 not on npm
```
Comment (jdlrobson) [inline openlibrary/plugins/openlibrary/js/index.js]: it's on npm so this comment is incorrect. Let's replace this with import 'jquery-validation'

File: package.json [hunk 2]
```diff
@@ -33,7 +33,7 @@
     },
     {
       "path": "static/build/all.js",
-      "maxSize": "144.3KB"
+      "maxSize": "145.77KB"
```
Comment (jdlrobson) [inline package.json]: Round up to 145.8KB for consistency with the other numbers and a bit of "wiggle room" :)

Comment (cbolgiano) [reply package.json]: Wiggle room is good.  Change has been made in most recent commit.

File: openlibrary/plugins/openlibrary/js/index.js (hunk 1 above)
Comment (cbolgiano) [reply openlibrary/plugins/openlibrary/js/index.js]: Will do.

File: vendor/js/jquery-validate/jquery.validate.js [hunk 3]
```diff
@@ -1,76 +1,104 @@
-/*
```
[the remark this answers is not in the document]
Comment (cbolgiano) [reply vendor/js/jquery-validate/jquery.validate.js]: Done.

[the remark this answers is not in the document]
Comment (cbolgiano) [reply vendor/js/jquery-validate/jquery.validate.js]: Forgot to save the npm dependency.  It is now included in the latest commit.

**Comment by jdlrobson** [comment]
Thank you!

**Pull request merged**